cdp_scrapers/README.md

```diff
@@ -1,12 +1,19 @@
-# Notes on `LegistarScraper`s
```

Comment:
General note: this README doesn't get rendered to docs I don't believe can we move it to the docs folder and do an `mdinclude`

Reply:
I added `legistar_scraper.rst` which does `.. mdinclude:: ../cdp_scrapers/README.md`. `legistar_scraper.rst` is then referenced in `index.rst`. Didn't think I can use `mdinclude` directive in an `md`?

Reply:
Correct! You cannot unforunately :(